components/FoodGroups/Draggable/characterspawner.tsx

```diff
@@ -0,0 +1,242 @@
+import React, { useState, useEffect } from 'react'
+import Draggable, { DRAGGING_STATE } from '@components/FoodGroups/Draggable'
+import { State_, StateDispatch } from '../types'
```

Comment:
Need to re-organise imports. It improves readability a lot.

A good approach is:
- have package imports first (eg: `import Link from 'next/link'`)
- then absolute imports next (eg: `import @components/something`)
- last will be the relative imports (eg: `import '../types'`)